src/modules/Movies/add_movie.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../Classes/movie'
+module AddMovie
+  def add_movie
+    print 'Movie name: '
+    name = gets.chomp
+    print 'Published date: '
+    publish_date = gets.chomp
+    print 'Movie is silent? [Y/N]'
+    silent = gets.chomp.upcase
+    case silent
+    when 'Y'
+      silent = true
+    when 'N'
+      silent = false
```

Comment:
You can simplify this block of code to one line.
like this: `silent = gets.chomp.upcase== 'y'`